src/atopile/kicad_plugin/common.py

```diff
@@ -91,3 +91,41 @@ def sync_footprints(
         target_fp.SetOrientation(source_fp.GetOrientation())
         target_fp.SetLayer(source_fp.GetLayer())
     return missing_uuids
+
+
+def find_anchor_footprint_group(group: pcbnew.PCB_GROUP) -> pcbnew.FOOTPRINT:
+    """Return anchor footprint with largest pin count in group: tiebreaker size"""
+    max_padcount = 0
+    max_area = 0
+    anchor_fp = None
+    items = group.GetItems()
+    fps = [item for item in items if isinstance(item, pcbnew.FOOTPRINT)]
+    for fp in fps:
+        if fp.GetPadCount() > max_padcount or (fp.GetPadCount()==max_padcount and fp.GetArea()>max_area):
+            anchor_fp = fp
+            max_padcount = fp.GetPadCount()
+            max_area = fp.GetArea()
+    return anchor_fp
+
+
+def find_anchor_footprint_board(board: pcbnew.BOARD) -> pcbnew.FOOTPRINT:
```

Comment:
We could potentially consolidate these functions? Something for the future!

Reply:
Nice, I just did this in the pushgroup fix PR